.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,99 @@
+name: Bug report
+description: Create a bug report to help us improve
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for making NewPipe better by reporting a bug. :hugs:
+
+        Please fill in as much information as possible about your bug so that we don't have to play "information ping-pong" but can help you immediately.
```

Comment:
```suggestion
        Please fill in as much information as possible about your bug so that we don't have to play "information ping-pong", and can help you immediately instead.
```